Source/santad/EventProviders/SNTEndpointSecurityManager.h

```diff
@@ -17,5 +17,14 @@
 #include "Source/common/SNTKernelCommon.h"
 #include "Source/santad/EventProviders/SNTEventProvider.h"
 
+#include <EndpointSecurity/EndpointSecurity.h>
+
+// Gleaned from https://opensource.apple.com/source/xnu/xnu-4903.241.1/bsd/sys/proc_internal.h
+const pid_t PID_MAX = 99999;
+
 @interface SNTEndpointSecurityManager : NSObject<SNTEventProvider>
+- (santa_vnode_id_t)vnodeIDForFile:(es_file_t *)file;
+
+@property(nonatomic) es_client_t *client;
```

Comment:
Can this be readonly here, then make it readwrite in SNTEndpointSecurityManager.mm?

Reply:
Good call